server/routes/share/scroll/[request].ts

```diff
@@ -59,7 +59,15 @@ const getData = async (userId: number) => {
         code: hatcheryTable.id,
       })
       .from(hatcheryTable)
-      .where(eq(hatcheryTable.user_id, userId)),
+      .where(
+        and(
+          eq(hatcheryTable.user_id, userId),
+          or(
+            eq(hatcheryTable.in_seed_tray, 1),
+            eq(hatcheryTable.in_garden, 1)
```

Comment:
this is fine but is just a little... misleading? you can use `true` here and drizzle will be able to work out what we mean and it's less mind-juggling